pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go

```diff
@@ -178,7 +177,6 @@ func TestSchedulingReconciler(t *testing.T) {
 			),
 			wantStatus: reconcileStatusContinue,
 		},
-		// TODO:(p0lyn0mial): fix me
 		/*{
```

Comment:
I think we have to fix these, or rewrite or prune from irrelevant tests.

Reply:
after all workspace scheduling changed a lot :)